openapi-spec/swagger.yaml

```diff
@@ -0,0 +1,1745 @@
+                type: object
+                required:
+                      - storages
+                additionalProperties: true
+                properties:
+                  storages:
+                    type: array
+                    title: The storages schema
+                    items:
+                      $ref: '#/components/schemas/StorageBackendResponse'
+        '401':
+          description: NotAuthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorSpec'
+        '403':
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorSpec'
+        '500':
+          description: An unexpected error occurred.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorSpec'
+    post:
+      tags:
+        - Storages
+      description: Register a storage device for management.
+      operationId: addStorageBackends
+      requestBody:
+        description: Inventory item to add
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/StorageBackendRegistry'
+      responses:
+        '200':
+          description: 'Accepted, items added to the infrastructure management'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/StorageBackendResponse'
+        '400':
+          description: BadRequest
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorSpec'
+        '401':
+          description: NotAuthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorSpec'
+        '403':
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorSpec'
+        '409':
+          description: An item already exists
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorSpec'
+        '500':
+          description: An unexpected error occured.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorSpec'
+  '/v1/storages/{storage_id}':
+    get:
+      tags:
+        - Storages
+      description: Get details of a storage device
+      operationId: GetStorageBackendbyID
+      parameters:
+        - name: storage_id
+          in: path
+          description: Database ID created for a storage backend .
+          required: true
+          style: simple
+          explode: false
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Storage backend list available
```

Comment:
Suggest to change to "Storage backend available"

Reply:
done